cylc/flow/loggingutil.py

```diff
@@ -202,10 +240,52 @@ def do_rollover(self):
             if self.FILE_NUM in header_record.__dict__:
                 # Increment log file number
                 header_record.__dict__[self.FILE_NUM] += 1
+                # strip the hard coded log number (1) from the log message
+                # replace with the log count number of that run.
+                # Note this is different from the log number in the file name.
```

Comment:
This reference to "log count number" that is different to the the "log number in the file name" is confusing to me

Reply:
I'll re-write it. This note basically highlights that the log number at the beginning of the file name is the cumulative log number, whereas the log number that appears inside the file is the number of logs that have been created for this 'start'. I will also change the logging statement to make it clear.